collaborative-plonk/src/round3.rs

```diff
@@ -517,7 +517,8 @@ pub mod tests {
     use std::{fs::File, io::BufReader};
 
     use ark_bn254::Bn254;
-    use circom_types::{groth16::witness::Witness, plonk::ZKey};
+    use circom_types::groth16::Witness;
```

Comment:
This witness type is also no longer `groth16` only, it  should live 1 lvl higher

Reply:
moved it